modules/tinymce/src/plugins/lists/test/ts/webdriver/DeleteTest.ts

```diff
@@ -61,4 +61,43 @@ describe('webdriver.tinymce.plugins.lists.DeleteTest', () => {
       TinyAssertions.assertContent(editor, initialContent);
     });
   });
+
+  context('Prevention', () => {
+    const preventionHook = TinyHooks.bddSetupLight<Editor>({
+      plugins: 'lists',
+      toolbar: 'numlist bullist',
+      indent: false,
+      setup: (editor: Editor) => {
+        editor.on('init', () => {
+          editor.on('keydown', (event) => {
+            event.preventDefault();
+          });
+        });
+      },
+      base_url: '/project/tinymce/js/tinymce'
+    }, [ Plugin ], true);
+
+    it('TINY-13276: Backspace in selection is preventable', async () => {
+      const editor = preventionHook.editor();
+
+      const initialContent = '<ol>' +
+        '<li>' +
+        'List 1' +
+        '</li>' +
+        '</ol>';
+
+      const expectedContent = '<ol>' +
+        '<li>' +
+        'List 1' +
+        '</li>' +
+        '</ol>';
+
+      editor.setContent(initialContent);
+      editor.undoManager.add();
+      TinySelections.setSelection(editor, [ 0, 0, 0 ], 2, [ 0, 0, 0 ], 5);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.backspace() ]);
```

Comment:
We dont need realkeys for this. You can check if the event was prevented by the isDefaultPrevented() on the event or make sure that the operation never happened.

We should avoid having more webdriver tests that will break. If you make a non web driver test then we can stick that into some other list delete related test file.

Reply:
This one does check that the event didn't happen by checking that no change occurred on the content. As for not making it a webdriver, trying the fake keys seems to not work right, not sure why.